nextJSTypeScript/Dockerfile

```diff
@@ -0,0 +1,139 @@
+# ARGUMENTS --------------------------------------------------------------------
+##
+# Board architecture
+##
+ARG IMAGE_ARCH=
+# For armv7 use:
+#ARG IMAGE_ARCH=arm
+
+##
+# Base container version
+##
+ARG BASE_VERSION=3-bookworm
+
+##
+# Application Name
+##
+ARG APP_EXECUTABLE=__change__
+
+##
+# Board GPU vendor prefix
+##
+ARG GPU=
+# ARGUMENTS --------------------------------------------------------------------
+
+
+
+# BUILD ------------------------------------------------------------------------
+FROM --platform=linux/${IMAGE_ARCH} \
+    torizon/wayland-base${GPU}:${BASE_VERSION} AS Build
```

Comment:
There is some specific reason you are using the `torizon/wayland-base` instead `torizon/debian`?

Reply:
I'm not sure about this.
Since Next.js has both a Server side and a client side, and the GUI is accessed through Chromium (typically), I thought that `torizon/wayland-base` would be necessary to support GPU-accelerated rendering.
But maybe Chromium container is enough to do this, and `torizon/debian` is the right choice.

Reply:
ok, I got it. Yeah, on my changes I turned back to `torizon/debian`, later we can set up the Chromium to run from docker-compose.yml